External/RingFamilies/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+if(RDK_USE_URF)
+
+include(ExternalProject)
+
+ExternalProject_Add(RingDecomposerLib_project
+    GIT_REPOSITORY https://github.com/rareylab/RingDecomposerLib.git
+    GIT_TAG master
```

Comment:
There shouldn't be a lot of changes to the RingDecomposerLib anymore, however, it is not guaranteed that future changes to the RingDecomposerLib won't break the RDKit build (especially if the changes are touching the cmake parts). I can create a tag on the current state of the RingDecomposerLib, if you want a frozen version. Your choice :)

Reply:
That would be great and definitely help with stability